packages/build/src/core/build.ts

```diff
@@ -579,3 +594,13 @@ const runBuild = async function ({
 
   return { stepsCount, netlifyConfig: netlifyConfigA, statuses, failedPlugins, timers: timersB, configMutations }
 }
+
+type Runtime = {
+  package: string
+  skipFlag: string
+}
+
+const supportedRuntimes: Record<string, Runtime> = {
```

Comment:
Are we sure we want to hardcode this in Netlify Build as opposed to moving it to, say, framework-info?

Reply:
Is it possible to modify netlifyConfig from framework-info?

Reply:
You don't need to modify the config from framework-info, but it might make sense for the data of which plugins are associated with each framework to live there.